scripts/qa_crowbarsetup.sh

```diff
@@ -3889,7 +3889,7 @@ function oncontroller_testsetup()
     wait_for 200 5 "openstack image show $imageid | grep status.*active" \
         "$image_name image to reach active state"
 
-    if [[ $want_ldap ]] ; then
+    if [[ $want_ldap = 1]] ; then
```

Comment:
Note: This is a workaround until https://github.com/SUSE-Cloud/automation/pull/1247 is tested and merged.